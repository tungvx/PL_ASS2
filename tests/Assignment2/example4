TEST=test
COMPLEX=more complex
LINES=multiple lines
%
This is a COMPLEX TEST that contains LINES.
Some MACRO can not be replaced.
LINE or LINESS are not correct macros.
This is also TESTreplaced.