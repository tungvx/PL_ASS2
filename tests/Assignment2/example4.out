This is a more complex test that contains multiple lines.
Some MACRO can not be replaced.
LINE or LINESS are not correct macros.
This is also testreplaced.